packages/qdrant-loader/tests/scripts/check_worker_pool.py

```
@@ -0,0 +1,122 @@
+    parser.add_argument("--worker-count", type=int, default=4)
+    parser.add_argument("--lease-seconds", type=int, default=60)
+    parser.add_argument("--same-source", action="store_true")
+    parser.add_argument("--fail-every", type=int, default=0)
+    args = parser.parse_args()
+
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(asctime)s | %(levelname)s | %(message)s",
+    )
+
+    db_path = Path(args.db).resolve()
```

Ruff (0.15.13)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)